website/docs/silk.net/experimental-feed.md

```diff
@@ -0,0 +1,66 @@
+---
+{
+    "TableOfContents": {
+        "Name": "Experimental Feed",
+        "Url": "experimental-feed.html"
+    }
+}
+---
+
+# Experimental Feed
+
+<?# Warning "The experimental feed is not recommended for use for anything beyond playing around with the new features. We don't officially support these builds as they may be unstable, and should not be used in production." /?>
+
+<?# Note "Changes in the experimental feed happen rapidly. We recommend joining the [Silk.NET Discord server](https://discord.gg/DTHHXRt) so that you can keep up with development." /?>
+
+## Configure your project
+
+Right now, your project should look something like this:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Silk.NET" Version="1.4.0" />
+  </ItemGroup>
+
+</Project>
+```
+
+In order to use the experimental feed, you must change this project file slightly. Add the following line to your project:
+
+```xml
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <!-- Add the experimental feed as a restore source -->
+    <RestoreSources>$(RestoreSources);https://dotnet.github.io/Silk.NET/nuget/experimental/index.json</RestoreSources>
+  </PropertyGroup>
+```
+
+## Install an experimental version
+
+Now .NET has access to the experimental feed, it's time to install an experimental package. Check for the latest version on our [GitLab Package Registry](https://gitlab.com/silkdotnet/Silk.NET/-/packages).
+
+At the time of writing, the latest version is `2.0.0-build97.0`. Now to install this package, you can use your IDE's built-in NuGet client, `dotnet package add`, or just modify the project file again like so:
+
+```xml
+  <ItemGroup>
+    <PackageReference Include="Silk.NET" Version="2.0.0-build97.0" />
+  </ItemGroup>
+```
+
+<?# Warning "Experimental Feed builds may be deleted without warning at the Silk.NET team's discretion." /?>
+
+Now you have access to bleeding-edge experimental builds of Silk.NET. Have fun!
+
+## GitHub Packages
+
+`https://dotnet.github.io/Silk.NET/nuget/experimental/index.json` shall always track the most up-to-date anonymous-access experimental feed. However, as an alternative, GitHub Packages can be used which requires authentication.
+
+To add this to your NuGet client, use URL `https://nuget.pkg.github.com/dotnet/index.json` in the configuration described in [the GitHub documentation](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry). The latest packages for this feed can be viewed [here](https://github.com/orgs/dotnet/packages?repo_name=Silk.NET).
```

Comment:
github packages was also broken, was this fixed?

Reply:
I’ve made an attempt!